src/vm/i386/stublinkerx86.cpp

```diff
@@ -4825,7 +4825,82 @@ VOID StubLinkerCPU::EmitSecureDelegateInvoke(UINT_PTR hash)
     EmitMethodStubEpilog(numStackBytes, SecureDelegateFrame::GetOffsetOfTransitionBlock());
 }
 
-#ifndef FEATURE_ARRAYSTUB_AS_IL
+//===========================================================================
+// Emits code to break into debugger
+VOID StubLinkerCPU::EmitDebugBreak()
```

Comment:
Moving this large block of code from one place to another makes the diff difficult to understand. Could you please leave the code where it was and just modify the ifdefs?